src/core/DBFileManager.cpp

```diff
@@ -0,0 +1,2 @@
+
+#include "DBFileManager.hpp"
```

Comment:
Did you mean to include this file combo?

Reply:
Sorry, no. Ill remove it from the commit.